manifests/l/Lenovo/UpdateRetriever/5.00.0096/Lenovo.UpdateRetriever.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Lenovo.UpdateRetriever
+PackageVersion: 5.00.0096
+PackageLocale: en-US
+Publisher: Lenovo
+PublisherUrl: https://www.lenovo.com
+PublisherSupportUrl: https://support.lenovo.com
+PrivacyUrl: https://www.lenovo.com/privacy
+Author: Lenovo
+PackageName: Lenovo Update Retriever
+PackageUrl: https://support.lenovo.com/solutions/ht037099
+License: (C) Copyright Lenovo 2021. All rights reserved.

```

```suggestion
License: Proprietary
```
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

---

Will do, thanks.